specification/oracle/Oracle.Database/operations/exadata/dbNodesOperations.tsp

```diff
@@ -0,0 +1,24 @@
+import "@azure-tools/typespec-azure-resource-manager";
+import "../../../models/exadata/dbNodes.tsp";
+
+using Azure.ResourceManager;
+using Autorest;
+using TypeSpec.Versioning;
+
+namespace Oracle.Database;
+
+@armResourceOperations
+interface DbNodes extends OracleReadOnlyResource<DbNode> {
+  // TODO: Swagger generator bug, cannot be extracted to interface
+  @example("./examples/dbNodes_listByParent.json", "List DbNodes by VM Cluster")
+  listByParent is ArmResourceListByParent<DbNode>;
```

Comment:
I am curious why this operation here are called listByParent but in openapi.json it's DbNodes_ListByCloudVmCluster ? are they really synced ?

Reply:
I found inconsistency only in DbNodes_listByParent example file and fixed that, but I see operationId in autogenerated openapi.json still same